guide/client_service/client_main.bal

```diff
@@ -0,0 +1,235 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/io;
+import ballerina/log;
+
+endpoint http:Client studentData {
+    url: " http://localhost:9292"
+};
+
+public function main(string... args) {
+    http:Request req = new;
+    int operation = 0;
+    while (operation != 6) {
+        // Print options menu to choose from.
+        io:println("Select operation.");
+        io:println("1. Add student");
+        io:println("2. View all students");
+        io:println("3. Delete a student");
+        io:println("4. Make a mock error");
+        io:println("5: Get a student's marks");
+        io:println("6. Exit \n");
+
+        // Read user's choice.
+        string choice = io:readln("Enter choice 1 - 5: ");
+        if (!isInteger(choice)){
+            io:println("Choice must be of a number");
+            io:println();
+            continue;
+        }
+
+        operation = check <int>choice;
+        // Program runs until the user inputs 6 to terminate the process.
+        if (operation == 6) {
+            break;
+        }
+        // User chooses to add a student.
+        if (operation == 1) {
+            addStudent(req);
+        }
+        // User chooses to list down all the students.
+        else if (operation == 2) {
+            viewAllStudents();
```

Comment:
Shall we re-format the if-else blocks merging else if/else with the previous line? Maybe:
```ballerina
         if (operation == 1) {
             // User chooses to add a student.
             addStudent(req);
         } else if (operation == 2) {             
            // User chooses to list down all the students.
             viewAllStudents();
         }
```

Reply:
Noted. I will make the changes.